### DISCUSSION TITLE
Setting permissions on volumes?

### DISCUSSION THREAD

**Discussion by dhnaranjo** [basecamp/kamal#134] [Q&A]
Hey, hi.

So I'm trying to have some storage persist across deploys to hold a SQLite db (litestream-backed) and I cannot sort out how to modify permissions on a directory on the host short of SSHing into it directly. Which like, is fine, but it'd be great if I could get the whole config running within mrsk. Am I missing some existing behavior or is this something y'all don't currently support?

If there isn't a good way to take care of it, maybe allow for user defined config scripts during server bootstrap'n?

**Comment by dhnaranjo** [comment]
What I could do is use the same user in both my app dockerfile and my ssh user, I suppose. That's probably how I'm meant to solve this? I'm not super great at docker'n, to be honest.

**Reply by tannakartikey** [reply]
You can use [Docker Volumes ](https://docs.docker.com/storage/volumes/)to store the persistent data. MRSK has `directories` config setting that allows you to set volumes on the container.
